src/services/exportInfoMap.ts

```diff
@@ -557,6 +558,18 @@ export function getExportInfoMap(importingFile: SourceFile | FutureSourceFile, h
             if (++moduleCount % 100 === 0) cancellationToken?.throwIfCancellationRequested();
             const seenExports = new Set<__String>();
             const checker = program.getTypeChecker();
+            if (isImportableSymbol(moduleSymbol, checker)) {
+                cache.add(
+                    importingFile.path,
+                    moduleSymbol,
+                    moduleSymbol.escapedName,

```

Comment:
I suspect this might be somewhat questionable solution here, I'm not fully sold on it myself either. cc @andrewbranch - you might have some opinions here (or suggestions for an improved fix? :P)